addons/web_editor/models/ir_qweb_fields.py

```diff
@@ -77,22 +77,24 @@ def _compile_directive_snippet(self, el, compile_context, indent):
 
         el.set('t-options', f"{{'snippet-key': {key!r}}}")
         view = self.env['ir.ui.view']._get(key).sudo()
-        name = view.name
+        name = el.attrib.pop('string', None) or view.name
```

Comment:
```suggestion
        name = el.attrib.pop('string', view.name)
```

No?

> Actually no: it works without adding the string attributes. But I add them anyway, so that the translatable text inventory finds them all.

How does it work without adding the string attribute if you do not do `_(view.name)` here? :thinking:  Can you explain? :) (including in the commit message)